data/language/en-GB.txt

```diff
@@ -3727,6 +3727,7 @@ STR_6276    :{RED}{STRINGID} has guests getting stuck, possibly due to invalid r
 STR_6277    :{WINDOW_COLOUR_2}Station index: {BLACK}{COMMA16}
 STR_6278    :Autosave amount
 STR_6279    :{SMALLFONT}{BLACK}Number of autosaves that should be kept
+STR_6280    :"{STRINGID}{OUTLINE}{TOPAZ}"{NEWLINE}{STRINGID}
```

Comment:
```suggestion
STR_6280    :{OUTLINE}{TOPAZ}“{STRINGID}”{NEWLINE}{STRINGID}
```
Always use proper curly quotes in localisation.

Reply:
Thanks for this! I didn't realize that there were curly quotes in this (as they appear extremely similar to standard quotes).